add-persistent-volume.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Do you need this on top of the file?
Also not sure about the filename

Reply:
suggestions?